src/components/spinner/Spinner.jsx

```diff
@@ -0,0 +1,34 @@
+import React,{Component} from 'react'
+import flywheel from './images/flywheel.gif'
+import pinwheel from './images/pinwheel.gif'
+import glowring from './images/glowring.gif'
+import turnsign from './images/turnsign.gif'
+
+
+
+const switchSpinner=(type,children)=>{
+    switch (type){
+        case 'flywheel':
+        return <div><img src={flywheel} alt="Fly wheel" /></div>
+
+        case 'pinwheel':
+        return <img src={pinwheel} alt="Pin Wheel" />
+
+        case 'glowring':
+        return <img src={glowring} alt="Glow Ring" />
+
+        case 'turnsign':
+        return <img src={turnsign} alt="Turn Sign" />
+    }
+}
+    
+
+const Spinner=(props)=>{
```

Comment:
Please de-structure your props for better code readability. e.g.
`const Spinner = ({type, children}) => {}`